frontend/src/containers/AddImage.tsx

```diff
@@ -217,6 +225,74 @@ function AddImage() {
                   />
                 </div>
 
+                <div className="grid-row">
+                  <div className="grid-col-9">
+                    <RadioButtons
+                      id="scalePct"
+                      name="scalePct"
+                      label={t("AddImageScreen.ImageSize")}
+                      hint={t("AddImageScreen.ImageSizeHint")}
+                      defaultValue={"auto"}
+                      register={register}
+                      onChange={handleImageScaleChange}
+                      options={[
+                        {
+                          value: "auto",
+                          label: t("AddImageScreen.AsUploaded"),
+                        },
+                        {
+                          value: "100%",
+                          label: t("AddImageScreen.100"),
+                        },
+                        {
+                          value: "75%",
+                          label: t("AddImageScreen.75"),
+                        },
+                        {
+                          value: "50%",
+                          label: t("AddImageScreen.50"),
+                        },
+                        {
+                          value: "25%",
+                          label: t("AddImageScreen.25"),
+                        },
+                      ]}
+                    />
+                  </div>
+                  <div className="grid-col-3 margin-top-10">
+                    <div style={{ position: "absolute", bottom: "0px" }}>
+                      {scalePct === "100%" && (
+                        <img
+                          src={`${process.env.PUBLIC_URL}/images/image-scale-100.svg`}
```

Comment:
recomendation: refactor URL resolution to a method and create only one image component

```
<img
    src={getScaleUrl(scalePct)}
    width="100%"
    height="auto"
  />
```

Reply:
Good recommendation. I changed it to use an inline function.